src/convertToObject.js

```diff
@@ -3,16 +3,29 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles
+ * (see an example in [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS properties
+ * (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styleObject = {};
+
+  sourceString.split(';')
+    .map(style => style.trim())
+    .filter(style => style)
+    .map(style => style.split(':')
+      .map(property => property.trim()))
+    .forEach(style => {
+      styleObject[style[0]] = style[1];
```

Comment:
It could be possible to avoid using **forEach** method. You can transfer line 25 to your map (line 23). Just like this **styleObject[style[0].trim()] = style[1].trim();** But it's all up to you

Reply:
agree